src/app/core/services/lti.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
```

Comment:
If you do an LTI module, I would expect more the service to be in it.
This would require to work differently, instead of having the layout service depending on it, the LTIService would require to "push" what's required on the core services... which is also cleaner as the core services would not know about LTI or any optional additions in the future.
I haven't read the code yet, so I don't know if that's possible, let me know your thoughts about that.

Reply:
I intended to put it in the LTI module at first, in the end I didn't because the layout service relies on it, and the layout service is declared (logically) at app-level.
I'll try reversing the dependency order, thanks to your comment I have another idea that will do well :wink:

Reply:
Fixed
Architecture is now much cleaner, however now there's a blink at startup in LTI mode: the left nav is displayed until the LTI module is being lazily fetched because the LTI component now is the one changing the app layout in its constructor.
Is that way ok or should I change it back ?